### PR TITLE
Removal of re injector from spinnMan and into FEC

### PR DESCRIPTION
**Pull request by alan-stokes**
this allows us to then insert the data gathering components into extra monitors cores. 

fec pull request depends upon this:
- [ ] https://github.com/SpiNNakerManchester/SpiNNFrontEndCommon/pull/215
- [ ] https://github.com/SpiNNakerManchester/SpiNNFrontEndCommon/pull/214
- [ ] https://github.com/SpiNNakerManchester/sPyNNaker/pull/427
- [ ] https://github.com/SpiNNakerManchester/SpiNNakerGraphFrontEnd/pull/70

### CODE REVIEW

**Comment by Christian-B** [comment]
.travis.yml still tries to build the c_models directory.

This needs to be removed from here and put where the c_models have gone.

**Comment by dkfellows** [comment]
You have some flake8 issues…

File: spinnman/messages/scp/enums/dpri_command.py [hunk 1]
```diff
@@ -4,12 +4,7 @@
 class DPRICommand(Enum):
     """ SCP Dropped Packet Reinjection Commands
     """
-    SET_ROUTER_TIMEOUT = (0, "Set the WAIT1 timeout of the router")
-    SET_ROUTER_EMERGENCY_TIMEOUT = (1, "Set the WAIT2 timeout of the router")
-    SET_PACKET_TYPES = (2, "Set the packet types to reinject")
-    GET_STATUS = (3, "Get the status of the reinjector")
-    RESET_COUNTERS = (4, "Reset the statistics counters")
-    EXIT = (5, "Exit the process")
+
```
Comment (dkfellows) [inline spinnman/messages/scp/enums/dpri_command.py]:
```
spinnman/messages/scp/enums/dpri_command.py:9:5: E303 too many blank lines (2)
```
Delete a blank line!

**Comment by Christian-B** [comment]
@dkfellows please fix travis.xml as c_model has moved.

**Pull request merged**